bonetrousle.py

```diff
@@ -0,0 +1,52 @@
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the bonetrousle function below.
+#
+
+def bonetrousle(n, k, b):
+    #
+    # Write your code here.
+    min_sum = b*(b+1)//2
+    max_sum = b*((k-b+1)+k)//2 
+    if n == min_sum:
+        boxes_list = [box for box in range(1, b+1)]
+    elif n == max_sum:
+        boxes_list = [box for box in range(k-b+1,k+1)]
+    elif min_sum < n > max_sum:

```

Comment:
I didn't know this was possible. For clarity, I would recommend using and:
if n > min_sum and n  > max_sum